src/clusterfuzz/_internal/tests/core/build_management/build_manager_test.py

```diff
@@ -26,6 +26,10 @@
 
 from clusterfuzz._internal.base import errors
 from clusterfuzz._internal.base import utils
+# Add this so some badly written code doesn't cause a test failure.
+# TODO(https://github.com/google/clusterfuzz/issues/4016): Fix this.
+from clusterfuzz._internal.bot.tasks.utasks import \
+    fuzz_task  # pylint: disable=unused-import
```

Comment:
Is this import alone adding some kind of env var somewhere? where exactly is it?

Reply:
I have no idea to be honest. Given that it's importing many other modules, it could be anywhere :facepalm:

Reply:
Yes I think it's adding some kind of env var.